From 1cd13d6b94c8cc432cce2d2811a79bfe58d57275 Mon Sep 17 00:00:00 2001
From: David Tardon <dtardon@redhat.com>
Date: Sat, 21 Sep 2013 14:23:38 +0000
Subject: WaE: ... should be initialized in the member init. list

Change-Id: I6c0d63b47f361b030481f58a6788b2ff4b0ee5be
---
diff --git a/src/lib/VSDXTheme.cpp b/src/lib/VSDXTheme.cpp
index 97be35d..8ea6cf6 100644
--- a/src/lib/VSDXTheme.cpp
+++ b/src/lib/VSDXTheme.cpp
@@ -31,7 +31,36 @@
 #include "VSDXMLTokenMap.h"
 #include "libvisio_utils.h"
 
+libvisio::VSDXVariationClrScheme::VSDXVariationClrScheme()
+  : m_varColor1()
+  , m_varColor2()
+  , m_varColor3()
+  , m_varColor4()
+  , m_varColor5()
+  , m_varColor6()
+  , m_varColor7()
+{
+}
+
+libvisio::VSDXClrScheme::VSDXClrScheme()
+  : m_dk1()
+  , m_lt1()
+  , m_dk2()
+  , m_lt2()
+  , m_accent1()
+  , m_accent2()
+  , m_accent3()
+  , m_accent4()
+  , m_accent5()
+  , m_accent6()
+  , m_hlink()
+  , m_folHlink()
+  , m_variationClrSchemeLst()
+{
+}
+
 libvisio::VSDXTheme::VSDXTheme()
+  : m_clrScheme()
 {
 }
 
diff --git a/src/lib/VSDXTheme.h b/src/lib/VSDXTheme.h
index bb8b7e9..15d152f 100644
--- a/src/lib/VSDXTheme.h
+++ b/src/lib/VSDXTheme.h
@@ -49,6 +49,8 @@ struct VSDXVariationClrScheme
   Colour m_varColor5;
   Colour m_varColor6;
   Colour m_varColor7;
+
+  VSDXVariationClrScheme();
 };
 
 struct VSDXClrScheme
@@ -66,6 +68,8 @@ struct VSDXClrScheme
   Colour m_hlink;
   Colour m_folHlink;
   std::vector<VSDXVariationClrScheme> m_variationClrSchemeLst;
+
+  VSDXClrScheme();
 };
 
 class VSDXTheme
--
cgit v0.9.0.2-2-gbebe
